client/verta/verta/deployment/update/rules.py

```diff
@@ -27,22 +27,31 @@ def _as_json(self):
 
     @staticmethod
     def _from_json(json_str):
-        rule_dict = json.loads(json_str)
+        return _UpdateRule._from_dict(json.loads(json_str))
+
+    @staticmethod
+    def _from_dict(rule_dict):
```

Comment:
That's a pretty good idea! Though now I realize it's awkward having `_as_json()` also return a dict, and `_from_json()` being basically useless.

I've filed VR-5602 to follow up on that.

Reply:
Yikes, more tech debts!
But good to know that _something_ works.

Reply:
Actually, if you don't mind, I'll remove from_json here too. I'll fix as_json later on.